spark/sql/dataframe.go

```diff
@@ -148,6 +148,14 @@ type DataFrame interface {
 	Rollup(ctx context.Context, cols ...column.Convertible) *GroupedData
 	// SameSemantics returns true if the other DataFrame has the same semantics.
 	SameSemantics(ctx context.Context, other DataFrame) (bool, error)
+	// Sample samples a data frame without replacement and random seed.
+	Sample(ctx context.Context, fraction float64) (DataFrame, error)
+	// SampleWithReplacement samples a data frame with random seed and with/without replacement.
+	SampleWithReplacement(ctx context.Context, withReplacement bool, fraction float64) (DataFrame, error)
+	// SampleSeed samples a data frame without replacement and given seed.
+	SampleSeed(ctx context.Context, fraction float64, seed int64) (DataFrame, error)
```

Comment:
```suggestion
	SampleWithSeed(ctx context.Context, fraction float64, seed int64) (DataFrame, error)
```

Reply:
the parameter `withReplacement` in the other cases already contains the "with" part in the name.